README.md

```diff
@@ -0,0 +1,37 @@
+# keyboard-only-outlines
+
+Disable outlines caused by navigation methods other than tab-navigation.
```

Comment:
`other than tab-navigation` -> `other than keyboard navigation (e.g.: tab).`

Reply:
You also forgot the README badges, check other repositories.

Reply:
I think "displayed" sounds better than "caused". What do you think? 

> Disable outlines displayed by navigation methods other than keyboard navigation (e.g.: tab).

Reply:
Agree

Reply:
Don't forget to update `package.json`. 😄

Reply:
I don't agree with this. Outlines aren't "displayed" by navigation methods, the navigation methods used may lead to outlines being displayed but are not the direct cause of it.

I suggest the following:

> Disable outlines displayed when using navigation methods other than keyboard navigation (e.g.: tab).

Reply:
@Areadrill Perfect 👌